test/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/fileupload/ScheduleOfMrnDocumentControllerSpec.scala

```diff
@@ -97,33 +91,67 @@ class ScheduleOfMrnDocumentControllerSpec extends FileUploadControllerSpec {
       "show the file upload failed page" when {
 
         "uploaded file size exceeds threshold" in {
-          def performAction(): Future[Result] =
+          val (session, _, _) = sessionWithClaimState(None)
```

Comment:
copy-pasta error - `sessionWithClaimState` should be something like `sessionWithScheduleOfMrnDocumentState` or if we don't want to have to remember doing that all the time when copying other tests specs then we can generalise this maybe and add it in the ControllerSpec class as a helper utility.